.github/workflows/push-tests.yml

```diff
@@ -222,16 +222,10 @@ jobs:
             **/Dockerfile
             requirements*.txt
             **/requirements*.txt
-      - name: Check if containers need to be rebuilt
-        if: steps.changed-files.outputs.any_changed == 'true'
-        run: |
-          echo "DOCKER_REPO=${DOCKER_REPO}" > .env.ecr
-          make local-rebuild-backend
-      - name: Run tests in docker-compose
+      - name: Run unit tests
         run: |
           echo "DOCKER_REPO=${DOCKER_REPO}" > .env.ecr
-          make local-init-test-data
-          make all-local-unit-test-backend
+          make redesign-unit-test
```

Comment:
why aren't we running these tests in the docker image? This would fix the gevent monkey wrench problem.

Reply:
They are run in Docker. Check the makefile rule.